### PR TITLE
feat: supporting new schema refactor

### PR DESCRIPTION
**Pull request by AlexsJones**
Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by AlexsJones** [comment]
https://github.com/k8sgpt-ai/k8sgpt/pull/1219

**Comment by matthisholleville** [comment]
You need to update your branch

**Review by matthisholleville — APPROVED** [review]

File: chart/operator/templates/k8sgpt-crd.yaml [hunk 1]
```diff
@@ -212,3 +212,5 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
+
+
```
Comment (JuHyung-Son) [inline chart/operator/templates/k8sgpt-crd.yaml:216]: 2 new lines?

**Review by JuHyung-Son — APPROVED** [review]
LGTM!

File: chart/operator/templates/k8sgpt-crd.yaml (hunk 1 above)
Comment (AlexsJones) [reply chart/operator/templates/k8sgpt-crd.yaml:216]: I think this is because I copied it over manually, ill remove

**Pull request merged**